visual_reasoning_models/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Downloaded datasets
```

Comment:
Why do we need two .gitignore?

Reply:
I find it cleaner that way. In any case, I would like to discuss with you about what's the standard way of downloading datasets in NeMo. Is there some kind of standard directory where all datasets go? Or maybe the user has to do it manually?